contrib/initialize_chains.py

```diff
@@ -102,7 +102,6 @@ def initialize_territories(volume_fraction, mapN, nchains, lattice="hcp", rs=Non
     else:
         # rs is the desired cell size of each square in lattice
         positions *= rs
```

Comment:
add back the comment before line 105 "# now set radius of sphere to be that of individual chain"